mongoose.c

```diff
@@ -3325,6 +3372,14 @@ static void open_local_endpoint(struct connection *conn) {
 
   conn->mg_conn.content_len = cl_hdr == NULL ? 0 : (int) to64(cl_hdr);
 
+#ifndef NO_AUTH
```

Comment:
where does this `#define` come from? if it is something we defined, we should change this to `#ifdef WITH_AUTH` instead of the double negation.

Reply:
I think it is part of upstream mongoose C library. 

https://github.com/Proemion/mongoose-cpp/blob/4f2db620bffa454a9b025263d93224d26d99f922/mongoose.c#L198

Reply:
ah, allright then.